Makefile.am

```diff
@@ -1 +1 @@
-SUBDIRS = lib vslib syncd
+SUBDIRS = meta lib vslib syncd player
```

Comment:
player seems to be counterpart of syncd. Maybe better to be syncd? saimetadata seems to be metadata checking library that can be used by others?